src/Uno.UI.Runtime.Skia.X11/X11KeyboardInputSource.cs

```diff
@@ -45,7 +45,7 @@ internal void ProcessKeyboardEvent(XKeyEvent keyEvent, bool pressed)
 			}
 
 			var symbols = System.Text.Encoding.UTF8.GetString(buffer, nbytes); // According to the docs, this should be ISO Latin-1 or ASCII
-			if (string.IsNullOrEmpty(symbols) || char.IsControl(symbols[0]))
+			if (string.IsNullOrEmpty(symbols) || (symbols != "\r" && char.IsControl(symbols[0])))
```

Comment:
It's quite unfortunate that we have a property on KeyEventArgs and KeyRoutedEventArgs that's only relevant to TextBox though 😕